src/blocks/utils/python.ts

```diff
@@ -23,9 +23,9 @@ import { ClassData, PythonData, organizeVarDataByType, VariableGettersAndSetters
 import { robotPyData } from './robotpy_data';
 import { externalSamplesData } from './external_samples_data';
 
-import * as pythonEnum from "../mrc_get_python_enum_value";
-import * as getPythonVariable from "../mrc_get_python_variable";
-import * as setPythonVariable from "../mrc_set_python_variable";
+import * as PythonEnum from "../mrc_get_python_enum_value";
```

Comment:
Why did these get changed from lower case to upper case when in other files you changed them from upper case to lower case?   I don't understand when it should start with lower and when with upper.

Reply:
I noticed some of yours were upper so I looked it up and Google's ai summary said that upper is preferred for "import * as".